storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -53,5 +53,25 @@ include::modules/persistent-storage-csi-vsphere-top-aware.adoc[leveloffset=+1]
 .Additional resources
 * https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-16422FF7-235B-4A44-92E2-532F6AED0923.html?hWord=N4IghgNiBcIC5gOYgL5A[VMware vSphere tag documenation]
 
+== vSphere encryption
+
+You can encrypt virtual machines (VMs) and persistent volumes (PVs) on {product-title} running on top of vSphere. This feature allows user-provisioned persistent storage to be encrypted over wire. 
```

Comment:
"user-provisioned" persistent storage word is ambiguous. It should be called - "dynamically provisioned PVs". Also encrypted over wire could be wrong. It implies certain encryption mechanism which we do not support.

Reply:
Done.